equipe.html

```diff
@@ -1,20 +1,68 @@
 ---
 title: La communauté
+additional_css: equipe
+additional_js:
+  - https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.2.1/Chart.bundle.min.js
+  - equipe
+hire:
+  id: recrutement
+  fullname: Vous ?
+  role: Développeu·r·se, chef·fe de produit numérique, coach lean, expert·e marketing digital…
+  link: /contact
+  avatar: /img/logo-generique-startup-carre.jpg
+  content: <b>Vous avez envie de nous rejoindre ? Contactez-nous !</b>
 ---
+{% capture build_date %}{{ site.time | date:"%Y-%m-%d" }}{% endcapture %}
+{% assign members = site.author | sort: 'start' | reverse %}
+
 <section class="ui four stackable doubling cards container">
 	<h2 class="ui divider horizontal">La communauté des Startups d'État</h2>
 
-	{% for person in site.author %}
-		{% unless person.end %}
+	{% for person in members %}
+		{% capture end_date_as_string %}{{ person.end }}{% endcapture %}
+		{% if end_date_as_string == '' or end_date_as_string > build_date %}
 			{% include author.html description=person %}
-		{% endunless %}
+			<!--{% increment members_count %}-->
+		{% endif %}
 	{% endfor %}
 
-	<h2 class="ui divider horizontal">Les ancien·ne·s</h2>
+	{% include author.html description=page.hire %}
 
-	{% for person in site.author %}
-		{% if person.end %}
+	<div id="effectifs">
+		<h2 class="ui divider horizontal">Évolution des effectifs</h2>
+		<canvas width="768" height="400">
+			Nous avons actuellement {% increment members_count %} membres.
```

Comment:
Oui, c'est du [fallback content](http://www.w3.org/TR/html5/scripting-1.html#the-canvas-element) pour l'accessibilité (lecteur d'écran, SEO, pas de JS, connectivité lente…).